tests/shell/shell_recorder/listtest.esr

```diff
@@ -16,4 +16,5 @@ RET: 0
 
 < kdb ls $Mountpoint
 
+< kdb rm /sw/kdb/current/plugins
```

Comment:
Please create an issue or add it to #401.

I am afraid you have to do import/export (currently not working with all plugins, see #51) or copy all namespaces separately (using `kdb cp`). Did you also try `kdb mv`?